Makefile

```diff
@@ -119,8 +122,9 @@ _output/bin/$(GOOS)/$(GOARCH)/$(BIN): build-dirs copy-astrolabe
 TTY := $(shell tty -s && echo "-t")
 
 shell: build-dirs
-	@echo "running docker: $@"
-	docker run \
+	@echo "running $(DOCKER): $@"
+	$(DOCKER) run \
+		--platform linux/amd64 \
```

Comment:
Can you use $(GOOS)/$(GOARCH) instead so that it is dynamically populated?

Reply:
Great catch - done.